plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemMaxDurability.java

```diff
@@ -0,0 +1,51 @@
+package com.denizenscript.denizen.objects.properties.item;
+
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.Material;
+import org.bukkit.inventory.meta.Damageable;
+import org.bukkit.inventory.meta.ItemMeta;
+
+public class ItemMaxDurability extends ItemProperty<ElementTag> {
+
+    // <--[property]
+    // @object ItemTag
+    // @name max_durability
+    // @input ElementTag
+    // @description
+    // Controls the maximum durability of an item.

```

Comment:
The meta for things should generally have more information than this. Give examples. Give limits. Give edge cases (the tag can be null - when/why?)